UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ExchangeSetForLargeFile.cs

```diff
@@ -61,23 +60,37 @@ public void WhenICallExchangeSetApiWithMultipleProductIdentifiers_ThenAMediaTxtF
 
         [Test]
         [Category("QCOnlyTest")]
-        public void WhenICallExchangeSetApiWithMultipleProductIdentifiers_ThenAnINFOFolderIsGenerated()
+        public void WhenICallExchangeSetApiWithMultipleProductIdentifiers_ThenAnINFOFolderWithFilesIsGenerated()
         {
+            string[] infoFolderFiles = { "AVCS-User-Guide.pdf", "ENC TandP NM status.pdf", "End-User-Licence-Agreement-for-ADMIRALTY-digital-data-services.pdf", "Important Information for AVCS users.pdf" };
```

Comment:
even if these files are always the same name it is better to place this test data in a central holder not in the test case code.

Reply:
updated. Please check..

Thanks,
Mayuresh Gawde